tests/services/cli_test.py

```diff
@@ -175,7 +175,7 @@ def tearDown(self):
 
         ('model delete foo',
          CLIENT.model.delete_model,
-         ["foo"],
+         ['da39a3ee5e6b4b0d3255bfef95601890afd80709'],
```

Comment:
From the comment above, do we have a test case for deleting both by handle and by name?

Reply:
Same as above, since the get_model by name or handle is already tested, we don't need to re test any methods.

Reply:
Okay.